web/src/app/core/resolvers/edit-project.resolver.ts

```diff
@@ -12,12 +13,22 @@ import { IProject, ProjectModel } from '@shared/models/index.model';
 })
 export class EditProjectResolver implements Resolve<IProject> {
 
+  /**
+   * Life cycle method
+   * @param authService AuthenticationService instance
+   * @param projectService ProjectService
```

Comment:
This one does not have `instance` after it, but I don't think it is needed, it does not add value to the docs. Sorry but I think it is best to remove it from everywhere

Reply:
Removed from all places